packages/react/tests/e2e-storybook/cypress/integration/Card/Card-Static.e2e.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import settings from 'carbon-components/es/globals/js/settings';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Defines the component variant path.
+ *
+ * @type {string}
+ * @private
+ */
+const _path = '/iframe.html?id=components-card--card-static';
+
+/**
+ * Defines the base card component selector.
+ *
+ * @type {string}
+ * @private
+ */
+const _selectorBase = `[data-autoid="${ddsPrefix}--card"]`;
+
+/**
+ * Defines the card element selectors.
+ *
+ * @type {Object.<string>}
+ * @private
+ */
+const _selectors = {
+  eyebrow: `${_selectorBase} .${prefix}--card__eyebrow`,
+  heading: `${_selectorBase} .${prefix}--card__heading`,
+  footer: `${_selectorBase} .${prefix}--card__footer`,
+  image: `${_selectorBase} .${prefix}--image img`,
+  copy: `${_selectorBase} .${prefix}--card__copy`,
+};
+
+/**
+ * Defines viewport dimensions to be used in tests.
+ *
+ * @type {number[]}
+ * @private
+ */
+const _viewportDimensions = [1280, 780];
+
+describe('Card | Static', () => {
+  beforeEach(() => {
+    cy.viewport(..._viewportDimensions);
+    cy.visit(_path);
+  });
+
+  it('should render eyebrow content', () => {
+    cy.get(_selectors.eyebrow).should('not.be.empty');
+  });
+
+  it('should render heading content', () => {
+    cy.get(_selectors.heading).should('not.be.empty');
+  });
+
+  it('should render footer content', () => {
+    cy.get(_selectors.footer).should('not.be.empty');
+  });
+
+  it('should render copy', () => {
+    cy.get(_selectors.copy).should('not.be.empty');
+  });
+
+  it('should not respond to a click ', () => {
+    let initialLocation;
+    cy.location('href')
+      .then(location => {
+        initialLocation = location;
+      })
+      .get(_selectorBase)
+      .click()
+      .location('href')
+      .then(location => {
+        expect(location).to.equal(initialLocation);
+      });
+  });
+
+  it('should render correctly in all themes', () => {
+    cy.wrap(['w', 'g10', 'g90', 'g100']).each(theme => {
```

Comment:
this is great! I like that you looped through them instead of creating multiple tests for the themes! 

One nitpick, can you label the white theme as `white` instead of just the `w`?

Reply:
```suggestion
    cy.wrap(['white', 'g10', 'g90', 'g100']).each(theme => {
```

Reply:
Admittedly, I stole this from @andy-blum 🙂 https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/7673/files#diff-a42d6ed5ec53b43667f81c546bfc457893f5157db7a2bddd00eef414126fc929R35-R48

@ariellalgilmore and I spoke about making it reusable by adding it as a custom Cypress command. Andy, do you want to do that in your PR?

Reply:
@jkaeser I could do that. I'll have to look up how though 😀

Reply:
@andy-blum let me know if i can help! There is a mockMastheadFooter command in there right now

Reply:
Just added the `carbonScreenshot` command [here](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/7673/commits/d76b1ba24bdfa67f378588672b9dfa4a7ba5594a)